packages/core/src/utils/date.ts

```diff
@@ -2,13 +2,26 @@ import format from "date-fns/format";
 import formatISO from "date-fns/formatISO";
 import { utcToZonedTime } from "date-fns-tz";
 
-export const formatDate = (date: Date) =>
-  formatISO(date, { representation: "date" });
-
-export const formatDateTime = (date: Date) =>
-  format(utcToZonedTime(date, "UTC"), `HH:mm 'on' yyyy-MM-dd`) + " (UTC)";
-
-export const formatActivityDate = (activityDate: string) => {
-  const date = new Date(activityDate);
-  return format(utcToZonedTime(date, "UTC"), `yyyy-MM-dd, HH:mm:ss`) + " (UTC)";
+/**
+ * Formats date objects to human-readable strings, in UTC.
+ * @param date Date object to format.
+ * @param type Value to pass to specify format. If none specified, defaults to "date-only".
+ * @returns
+ */
+export const formatDate = (
+  date: Date,
```

Comment:
```suggestion
 /* Date object to format. */
  date: Date,
/* Value to pass to specify format. If none specified, defaults to "date-only". */
  type: "date-only" | "date-first" | "time-first" = "date-only"
```
You can also comment on you params this way. Produces the same result as the JSDOC

Reply:
Didn't know that, thank you!